adhocracy4/polls/views.py

```diff
@@ -41,6 +41,7 @@ def get_context_data(self, **kwargs):
         if self.object is not None:
             context['question_list'] = \
                 self.object.questions.annotate_vote_count().all()
```

Comment:
We could then also remove question_list from the context as it is not used anymore..